Documentation/tutorials/saving-and-resuming-tracks.md

```diff
@@ -0,0 +1,58 @@
+# Saving and Resuming a Trial
+
+This tutorial will walk you through the process of saving and resuming a trail. This is useful if you want to save your work and resume it later, or if you want to share your work with someone else.
+This tutorial assumes that you have already completed the [](./loading-and-tracking.md) tutorial or have otherwise loaded and tracked data in AutoscoperM.
+
+## Saving a Trial
+
+### File Structure
+
+In order to automatically load tracks when a trial is loaded in AutoscoperM, the tracks must be saved in a specific file structure. This is the same file structure that is shown in the [](./custom-data.md#automatic-filter-and-tracking-data-loading) section of the [](./custom-data.md) tutorial.
+
+
+The file structure is as follows:
+
+```
+trial name
+├── radiographs
+|   └── ...
+├── volumes
+|   └── ...
+├── calibration
+|   └── ...
+├── xParamters
+|   └── control_settings.vie
+├── Tracking
+|   ├── {trial name}_bone1.tra
+|   ├── {trial name}_bone2.tra
+|   ├── ...
+|   └── {trial name}_bonek.tra
+└── trial name.cfg
+```
+
+### Saving a Trial Configuration
+
+To save a trial configuration, click the `File` menu and select `Save a Trial` or press `Ctrl+Shift+S` on your keyboard. This will open a dialog box that will allow you to select a location to save the trial configuration. The trial configuration will be saved as a `.cfg` file.
+
+
+### Saving Tracks
+
+Inorder for tracks to be automatically loaded when a trial is loaded, tracks must be saved in the default configuration. For more info on the different track saving options, see the [](../user-interface.md#importexport-tracking-options) section of the User Interface page.
+
+For each bone you would like to save tracks for, select the bone in the `Volumes` panel (see the [](../user-interface.md) for more information). Once you have the desired bone selected, click the `File` menu and select `Save Tracking`, press `Ctrl+S` on your keyboard, or press the `Save Tracking` button on the [](../user-interface.md#toolbar). This will open a dialog box that will allow you to select a location to save the tracks. The tracks will be saved as a `.tra` file. Ensure that you are saving the tracks in the `Tracking` subfolder of the trial you are working on.
+
+### Saving Filter Settings
+
+Inorder for filter settings to be automatically loaded when a trial is loaded, filter settings must be saved into the default location. For more info on filters, see the [](./filters.md) tutorial. Ensure that you are saving the filter settings in the `xParameters` subfolder of the trial you are working on, and that the file is named `control_settings.vie`.
+
+```{note}
+The current version of Autoscoper only supports loading one settings file for all of the cameras. If you have different filter settings for different cameras, you will need to manually load the settings for each camera after loading the trial.
```

Comment:
If not already done, could we create an issue to describe the problem ?

Reply:
I'm not sure this is an issue but @amymmorton could confirm, I believe the user would want to only load one set of filters across all cameras.

I included this line to inform the user just in case they expected Autoscoper to load a unique file for each camera. I think the wording could be updated.

Reply:
There are instances when the user would have a different filter set for the radiograph per camera (is more or less tissue was in the FOV - think anterio-posterior knee vs medial-lateral) much more meat in one thatn the other and often one requires dif xray kv,ma than the other / one cam view has better contrast than the other.

At default launch no reason for two, but keep the option for specific users?

Reply:
Opened an issue for this, see #150